### ISSUE TITLE
Dropdown with MultiSelect & Required doesn't read "Required" with narrator

### ISSUE THREAD

**Issue by jongonzamsft** [microsoft/fluentui#15565]
### Environment Information

- **Package version(s)**: 7.128.1
- **Browser and OS versions**: Edge & Windows 10

### Describe the issue:

When the dropdown is set to multiselect and required, narrator does not properly read the input as required. Since the control is set to role="button" it isn't possible to also have it set to aria-required="true". 

#### Actual behavior:

Narrator reads the label and announced it has a popup.

#### Expected behavior:

Narrator should read the label and announce that the control is required

**Comment by paulgildea** [comment]
@jongonzamsft Thanks for the issue.

@smhigley Thoughts on this one?
